R/metadata.R

```diff
@@ -41,20 +47,31 @@
 #' imeta(
 #'  "foo.rds",
 #'  "data_object",
-#'  operations = list(
-#'     operation = "add",
-#'     attribute = "foo",
-#'     value = "bar",
-#'     units = "baz"
-#'     )
+#'  operations =
+#'   list(
+#'    list(operation = "add", attribute = "foo", value = "bar", units = "baz")
+#'  )
 #' )
 #'
+#' # `operations` can contain multiple tags supplied as a `data.frame`
+#' imeta(
+#'   "foo.rds",
+#'   "data_object",
+#'   operations = data.frame(
+#'     operation = c("add", "add"),
+#'     attribute = c("foo2", "foo3"),
+#'     value = c("bar2", "bar3"),
+#'     units = c("baz2", "baz3")
+#'  )
+#' )
+#'
+#' # or again as a list of lists
 #'imeta(
 #'  "foo.rds",
 #'  "data_object",
 #'  operations = list(
-#'    list(operation = "add", attribute = "foo2", value = "bar2"),
-#'    list(operation = "add", attribute = "foo3", value = "bar3")
+#'    list(operation = "add", attribute = "foo4", value = "bar4", units = "baz4"),
+#'    list(operation = "add", attribute = "foo5", value = "bar5", units = "baz5")
```

Comment:
I would actually have at least one of them without units to show that that is possible as well.

Reply:
I am thinking now that supplying of the different types to `operations` should be the most important message of the examples. Hence keeping the elements/columns of the `list` or `dataframe` the same.  It could be extended to include also examples of varying the elements. Op, perhaps, leave it up to the user to experiment with that?

Reply:
Or having a vignette about this...
But I would still have at least one of the examples not use units, because if all of them do, it can give the message that rirods only accepts AVUs with explicit units.

Reply:
I have added an additional example without units